features/volgindicaties.feature

```diff
@@ -0,0 +1,154 @@
+      Gegeven de persoon met burgerservicenummer "999993653" is NIET ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999993653"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999993653" en geen einddatum
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "999993653" naar het GBA-V 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een persoon met einddatum in de toekomst
+      Gegeven de persoon met burgerservicenummer "999993483" is NIET ingeschreven in deze gemeente
+      En er is geen enkele volgindicatie gezet voor de persoon met burgerservicenummer "999993483"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999993483" met einddatum "2021-03-15"
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "999993483" naar het GBA-V
+
+   Scenario: Een gebruiker zet een volgindicatie op een persoon die al gevolgd wordt door een andere gebruiker
+      Gegeven de persoon met burgerservicenummer "999990457" is NIET ingeschreven in deze gemeente
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999990457" en einddatum "2021-06-30"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999990457" en geen einddatum
+      Dan stuurt de applicatie GEEN Ap01 bericht met burgerservicenummer "999990457" naar het GBA-V
+
+   Scenario: Een gebruiker zet een volgindicatie op een persoon die niet meer gevolgd wordt door een andere gebruiker
+      Gegeven de persoon met burgerservicenummer "999992855" is NIET ingeschreven in deze gemeente
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999992855" en einddatum "2021-03-01"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992855" en geen einddatum
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "999992855" naar het GBA-V
+    
+   Scenario: Een gebruiker verlengt de einddatum van de volgindicatie
+      Gegeven de persoon met burgerservicenummer "999992806" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999992806" en einddatum "2021-03-15"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992806" met einddatum "2021-06-30"
+      Dan stuurt de applicatie GEEN Ap01 bericht met burgerservicenummer "999992806" naar het GBA-V
+
+   Scenario: Een gebruiker wijzigt de einddatum van de volgindicatie naar vandaag
+      Gegeven de persoon met burgerservicenummer "999994220" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999994220" en einddatum "2021-03-15"
```

Comment:
Deze twee gegevens:

- En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999994220" en einddatum "2021-03-15"
- En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999994220"

kunnen ook als volgt worden genoteerd:

En de persoon met burgerservicenummer heeft de volgende volgindicaties
| gebruiker | einddatum  |
| A              | 2021-03-15 |

Dan hoeft er niet nog een gegeven te worden opgenomen dat de persoon geen andere volgindicaties heeft. En bij meerdere volgindicaties kan het als volgt:

En de persoon met burgerservicenummer heeft de volgende volgindicaties
| gebruiker | einddatum  |
| A              | 2021-03-15 |
| B              | geen            |
| C              | 2021-12-31 |

Reply:
Maakt dat het duidelijker? In dit scenario heb ik maximaal twee gebruikers plus eventueel een regel dat er geen anderen zijn. Dat is toch niet zoveel groter of moeilijker dan het definiëren van een tabel?

Reply:
Voor mij is het duidelijker als je zegt een persoon heeft de volgende volgindicaties, in plaats van persoon heeft een indicatie en (nog) een indicatie en geen andere indicatie. Maar hoeft voor mij niet aangepast te worden

Reply:
ik laat het zo voor nu